<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1187</width>
    <height>832</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>15</pointsize>
   </font>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Fraggler GUI</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QFrame" name="frame_1">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_description">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <bold>true</bold>
             <underline>false</underline>
            </font>
           </property>
           <property name="text">
            <string>Fraggler GUI</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_icon">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>[ICON]</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_3">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,8">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="selectFileButton">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="toolTip">
                <string>Select a single FSA file for processing</string>
               </property>
               <property name="text">
                <string>Select File</string>
               </property>
               <property name="icon">
                <iconset theme="QIcon::ThemeIcon::WindowNew"/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="selectDirectoryButton">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="toolTip">
                <string>Select a directory that contains FSA files for bulk processing</string>
               </property>
               <property name="text">
                <string>Select Directory</string>
               </property>
               <property name="icon">
                <iconset theme="QIcon::ThemeIcon::DocumentOpen"/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QListWidget" name="fileListWidget"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3" stretch="6,3">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QWebEngineView" name="webEngineView">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>400</height>
              </size>
             </property>
             <property name="url">
              <url>
               <string>about:blank</string>
              </url>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="sizeConstraint">
              <enum>QLayout::SizeConstraint::SetMaximumSize</enum>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <property name="sizeConstraint">
                <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_6">
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_6">
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QLabel" name="typeLabel">
                     <property name="font">
                      <font>
                       <pointsize>10</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string>Type</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QComboBox" name="typeComboBox">
                     <property name="font">
                      <font>
                       <pointsize>10</pointsize>
                      </font>
                     </property>
                     <property name="toolTip">
                      <string>Fraggler area or fraggler peak.</string>
                     </property>
                     <property name="currentText">
                      <string/>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="ladderLabel">
                     <property name="font">
                      <font>
                       <pointsize>10</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string>Ladder</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QComboBox" name="ladderComboBox">
                     <property name="font">
                      <font>
                       <pointsize>10</pointsize>
                      </font>
                     </property>
                     <property name="toolTip">
                      <string>Which ladder to use.</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="peakAreaModelLabel">
                     <property name="font">
                      <font>
                       <pointsize>10</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string>Peak Finding Model</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QComboBox" name="peakModelComboBox">
                     <property name="font">
                      <font>
                       <pointsize>10</pointsize>
                      </font>
                     </property>
                     <property name="toolTip">
                      <string>Which peak finding model to use</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="outputLabel">
                     <property name="font">
                      <font>
                       <pointsize>10</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string>Output Folder Name</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="outputFolderInput">
                     <property name="font">
                      <font>
                       <pointsize>10</pointsize>
                      </font>
                     </property>
                     <property name="toolTip">
                      <string>Output folder name (must be unique)</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_9">
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QLabel" name="sampleChannelLabel">
                     <property name="font">
                      <font>
                       <pointsize>10</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string>Sample Channel</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="sampleChannelInput">
                     <property name="font">
                      <font>
                       <pointsize>10</pointsize>
                      </font>
                     </property>
                     <property name="toolTip">
                      <string>Which sample channel to use. E.g. 'DATA1', 'DATA2'...</string>
                     </property>
                     <property name="inputMask">
                      <string/>
                     </property>
                     <property name="text">
                      <string>DATA1</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="peakStartLabel">
                     <property name="font">
                      <font>
                       <pointsize>10</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string>Peak Start</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="peakStartInput">
                     <property name="font">
                      <font>
                       <pointsize>10</pointsize>
                      </font>
                     </property>
                     <property name="toolTip">
                      <string>Where to start searching for peaks in basepairs</string>
                     </property>
                     <property name="text">
                      <string>115</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="minRatioLabel">
                     <property name="font">
                      <font>
                       <pointsize>10</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string>Min. Ratio to Allow Peak</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="minPeakRatioInput">
                     <property name="font">
                      <font>
                       <pointsize>10</pointsize>
                      </font>
                     </property>
                     <property name="toolTip">
                      <string>Minimum ratio of the lowest peak compared to the highest peak in the assay</string>
                     </property>
                     <property name="text">
                      <string>0.15</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="customPeaksLabel">
                     <property name="font">
                      <font>
                       <pointsize>10</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string>Custom Peaks (Optional)</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="customPeaksInput">
                     <property name="font">
                      <font>
                       <pointsize>10</pointsize>
                      </font>
                     </property>
                     <property name="toolTip">
                      <string>CSV Filename with custom peaks to find</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_4">
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QLabel" name="minDistanceBetweenPeaksLabel">
                     <property name="font">
                      <font>
                       <pointsize>10</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string>Min. Distance Between Peaks</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="minDistBetweenPeaksInput">
                     <property name="font">
                      <font>
                       <pointsize>10</pointsize>
                      </font>
                     </property>
                     <property name="toolTip">
                      <string>Minimum distance between size standard peaks</string>
                     </property>
                     <property name="text">
                      <string>30</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="minDistSizeStandardLabel">
                     <property name="font">
                      <font>
                       <pointsize>10</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string>Min. Height Size Standard Peaks</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="minHeightPeakInput">
                     <property name="font">
                      <font>
                       <pointsize>10</pointsize>
                      </font>
                     </property>
                     <property name="toolTip">
                      <string>Minimum height of peaks in sample data</string>
                     </property>
                     <property name="text">
                      <string>100</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="minPeakHeightLabel">
                     <property name="font">
                      <font>
                       <pointsize>10</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string>Min. Height Sample Peaks</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="minPeakHeightInput">
                     <property name="font">
                      <font>
                       <pointsize>10</pointsize>
                      </font>
                     </property>
                     <property name="toolTip">
                      <string>Minimum height of peaks in sample data</string>
                     </property>
                     <property name="text">
                      <string>300</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="distBetweenAssaysLabel">
                     <property name="font">
                      <font>
                       <pointsize>10</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string>Min. Distance Between Assays</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="minDistAssaysInput">
                     <property name="font">
                      <font>
                       <pointsize>10</pointsize>
                      </font>
                     </property>
                     <property name="toolTip">
                      <string>Minimum distance between assays in a multiple assay experiment</string>
                     </property>
                     <property name="text">
                      <string>15</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QPushButton" name="runFragglerButton">
                 <property name="font">
                  <font>
                   <pointsize>10</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>Run Fraggler</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QListWidget" name="listWidget">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>8</pointsize>
                </font>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1187</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_loadFSAFile"/>
    <addaction name="actionLoad_FSA_Directory"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionToggle_Dark_Light"/>
    <addaction name="action_about"/>
    <addaction name="actionZoom_Out"/>
    <addaction name="actionZoom_In"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
  </widget>
  <action name="action_loadFSAFile">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Load FSA File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionToggle_Dark_Light">
   <property name="text">
    <string>Toggle Dark/Light Mode</string>
   </property>
  </action>
  <action name="actionLoad_FSA_Directory">
   <property name="text">
    <string>Load FSA Directory</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="toolTip">
    <string>Zoom Out on Fraggler Report</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="toolTip">
    <string>Zoom In on Fraggler Report</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebEngineWidgets/QWebEngineView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
